.github/pull_request_template.md

```diff
@@ -0,0 +1,10 @@
+__Jira ticket__: https://broadworkbench.atlassian.net/browse/[ticket_number]
+
+## Addresses
+
+## Summary of changes
+
+## Testing Strategy
+
+<!-- Reminder -->
+<!-- Two team members from the Data Custodian Journeys team will be automatically assigned to review this pull request. If you otherwise have two reviewers, you do not need to wait for their review. -->
```

Comment:
Would you mind tweaking this a bit further?

The need for this comment comes from the fact that this is a repo where (presently) both DCJ and DE teams are actively working.  The current CODEOWNERS settings have both teams set as owners, so the flip side is that DE team gets requested to review DCJ PRs too. Our repository norm is that we don't need to wait on the other team's approval to merge unless explicitly requested, in either direction.

Reply:
Ah, I didn't realize that we were getting randomly assigned to PRs as well. 
I made a slight tweak to the comment... I'm not sure how much specific detail I want to add to the comment. What do you think of the new version?

Reply:
I think that's closer, and in the name of not letting perfect be the enemy of good would elect to merge it as-is :-) I do agree with the instinct not to overexplain in the comment, as that would be liable to get out of date.  We can always adjust the comment down the road if we want.